### PR TITLE
Twig latest version of extension so we can use localizeddate.

### PR DESCRIPTION
**Pull request by bartdc**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/migrations": "^1.4",
         "doctrine/doctrine-migrations-bundle": "^1.1",
-        "twig/extensions": "^1.3",
+        "twig/extensions": "^1.5.1",
```
Comment (carakas) [inline composer.json]: @bartdc I think the lock file is still missing in this pr

Comment (bartdc) [reply composer.json]: This file didn't change, how can I push it?

Comment (StijnVrolijk) [reply composer.json]: @bartdc the file should have changed once you've run `composer install`

Comment (jonasdekeukelaere) [reply composer.json]: @bartdc have you been able to look at this?

Comment (bartdc) [reply composer.json]: The composer.json file didn't change after composer install. Any tips?

Comment (jonasdekeukelaere) [reply composer.json]: How did you update the package? You should either 
- change composer file and do `composer update twig/extensions`
- or just do `composer require twig/extensions:1.5.1`

Comment (StijnVrolijk) [reply composer.json]: @bartdc the `composer.json` file shouldn't change, it's the `composer.lock` file that should change.

**Comment by jonasdekeukelaere** [comment]
See https://github.com/sumocoders/Framework/pull/230

**Pull request closed without merging**